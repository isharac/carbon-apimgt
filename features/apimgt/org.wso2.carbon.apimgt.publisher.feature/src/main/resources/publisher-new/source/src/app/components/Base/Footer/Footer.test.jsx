import React from 'react';
import Footer from './Footer';

describe('Test scenarios for <Footer/> component', () => {
    test('should match with the existing snapshot', () => {
        const wrapper = renderer.create(<Footer />).toJSON();
        console.log(wrapper);
        expect(wrapper).toMatchSnapshot();
    });
});
