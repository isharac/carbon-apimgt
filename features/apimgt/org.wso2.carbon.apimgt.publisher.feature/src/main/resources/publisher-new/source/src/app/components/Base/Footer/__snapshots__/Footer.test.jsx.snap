// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test scenarios for <Footer/> component should match with the existing snapshot 1`] = `
<footer
  className="Footer-footer-1"
>
  <p
    className="MuiTypography-root-2 MuiTypography-body1-11 MuiTypography-noWrap-28"
  >
    <span>
      WSO2 APIM v3.0.0 | Â© 2019 WSO2 Inc
    </span>
  </p>
</footer>
`;
